src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,235 @@
+    <attribute side="server" code="0x0026" define="NEXT_CHARGE_TARGET_SOC"      type="INT8U"                      writable="false" default=""       optional="true"  isNullable="true">NextChargeTargetSoc</attribute>
+    <attribute side="server" code="0x0027" define="APPROX_EV_EFFICIENCY"        type="INT16U"                     writable="true"  default=""       optional="true"  isNullable="true">ApproxEvEfficiency</attribute>
+    <attribute side="server" code="0x0030" define="STATE_OF_CHARGE"             type="Percent" min="0" max="100"  writable="false" default=""       optional="true"  isNullable="true">StateOfCharge</attribute>
+    <attribute side="server" code="0x0031" define="BATTERY_CAPACITY"            type="INT32S"                     writable="false" default=""       optional="true"  isNullable="true">BatteryCapacity</attribute>
+    <attribute side="server" code="0x0032" define="VEHICLE_ID"                  type="CHAR_STRING"                writable="false" default=""       optional="true"  isNullable="true" length="32" >VehicleId</attribute>
+
+    <attribute side="server" code="0x0040" define="SESSION_ID"                  type="INT32U"                     writable="false" default="0x0"    optional="true"  isNullable="false">SessionId</attribute>
+    <attribute side="server" code="0x0041" define="EVENT_SEQUENCE_NUMBER"       type="INT16U"                     writable="false" default="0x0001" optional="true"  isNullable="false">EventSequenceNumber</attribute>
+    <attribute side="server" code="0x0042" define="SESSION_DURATION"            type="elapsed_s"                  writable="false" default="0x0"    optional="true"  isNullable="false">SessionDuration</attribute>
+    <attribute side="server" code="0x0043" define="SESSION_ENERGY_CHARGED"      type="INT32S"                     writable="false" default="0x0"    optional="true"  isNullable="false">SessionEnergyCharged</attribute>
+    <attribute side="server" code="0x0044" define="SESSION_ENERGY_DISCHARGED"   type="INT32S"                     writable="false" default="0x0"    optional="true"  isNullable="false">SessionEnergyDischarged</attribute>
+
+    <command source="client" code="0x01" name="Disable" optional="false">
+      <description>Stops the power flow between the EV and the EVSE.</description>
+    </command>
+
+    <command source="client" code="0x02" name="EnableCharging" optional="false">
+      <description>Allows the EVSE to charge the EV for the specified time within the limits specified in the fields of the command.</description>
+      <arg name="EnableChargeTime"     type="elapsed_s"                    optional="false"  isNullable="true"/>
+      <arg name="MinimumChargeCurrent" type="INT32U"   min="0" max="80000" optional="false"  isNullable="false"/>
```

Comment:
spec does not specify a min or a max for this or below.
Also min = 0 for a unsigned type is not useful.

Reply:
In the data type it says:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7813/files#:~:text=%5B%5Bref_EVSECurrentLimitType%2C%20EVSECurrentLimitType,0%2D80A)

=== EVSECurrentLimitType Type
This data type is derived from uint32.
This represents an EVSE current limit in mA. EVSE current can be advertised to
an EV in 0.6A steps.
The range of values permitted is between 0-80000mA (i.e. 0-80A).

I'll remove the min=0.